axelor-account/src/main/resources/views/PaymentSession.xml

```diff
@@ -211,6 +211,11 @@
       method="searchEligibleTerms"/>
   </action-method>
 
+  <action-group name="action-group-payment-session-search-eligible-terms">
+    <action name="save" if="!id"/>
```

Comment:
Please remove if="!id", the form must always be saved to fetch the updated payment session inside the controller